### ISSUE TITLE
Also hide parent container if all items are hidden in a filter

### ISSUE THREAD

**Issue by andyduo** [patrickkunka/mixitup#491]
I have a need to also hide the parent container if there are no .mix items showing. My code looks like this:

```html
<div class="container">
  <div class="app-glossary-item-wrapper">
    <div class="itemKey">Cars</div>
    <a class="mix cars">Some link text</a>
    <a class="mix cars">Some link text</a>
  </div>
  <div class="app-glossary-item-wrapper">
    <div class="itemKey">Bikes</div>
    <a class="mix cars">Some link text</a>
    <a class="mix cars">Some link text</a>
  </div>
</div>
```

So if all items are filtered out it also hides either the .itemKey heading or the .app-glossary-item-wrapper 

I hope this makes sense?

Thank you!

**Comment by patrickkunka** [comment]
Hi,

This isn't possible out of the box with MixItUp.

Typically, all target elements in a container must be adjacent siblings of the same parent, so I'm not sure if your markup structure will actually work.

However, if you wanted to hide or show parts of the DOM using custom javascript in reaction to a certain filter result, you could do that using the `onMixEnd` callback function, and the state object which is provided to that callback as the first argument, which will give you information about how many items are shown, or whether or not the mixer is in a "failed" state (i.e. no results found).

Be aware however that if the container or a parent element of any target is hidden at the start of an animation, this will most likely break the geometry calculations needed for the animations when you filter the items back in.

In conclusion, I'm pretty sure what you want to do won't be possible without breaking animations, but I would recommend having a play around with the `onMixEnd` callback anyway.

Hope that helps.
